_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -922,27 +1050,65 @@ accounts (e.g. dev, stage, and production) where your applications run.
 
 [[iam_roles_for_instances]]
 === Use IAM roles for EC2 instances
 All Gruntwork modules that require AWS API access use roles rather than an IAM user with static API credentials for
 authentication. For example:
 
-* link:https://github.com/gruntwork-io/module-server/blob/master/modules/single-server/main.tf[`module-server`]
+* link:https://github.com/gruntwork-io/terraform-aws-server/blob/master/modules/single-server/main.tf[`terraform-aws-server`]
  is used to manage a single EC2 instance with an IAM role attached.
-* link:https://github.com/gruntwork-io/module-asg[`module-asg`] applies IAM roles to instances in auto-scaling
+* link:https://github.com/gruntwork-io/terraform-aws-asg[`terraform-aws-asg`] applies IAM roles to instances in auto-scaling
  group.
 * link:https://github.com/gruntwork-io/terraform-aws-eks/blob/master/modules/eks-cluster-workers/main.tf[`terraform-aws-eks`]
  uses IAM roles for EKS cluster workers.
-* link:https://github.com/gruntwork-io/module-ecs/tree/master/modules/ecs-cluster[`ecs-cluster`] creates IAM
+* link:https://github.com/gruntwork-io/terraform-aws-ecs/tree/master/modules/ecs-cluster[`ecs-cluster`] creates IAM
  roles for ECS instances
-* link:https://github.com/gruntwork-io/package-lambda/tree/master/modules/lambda[`lambda`] creates IAM
+* link:https://github.com/gruntwork-io/terraform-aws-lambda/tree/master/modules/lambda[`lambda`] creates IAM
  roles for Lambda functions
 
 Use these modules whenever possible. You should always use IAM roles in your own modules any time you need to provide access to the AWS API. Using static API credentials should be avoided whenever possible.
 
+[[cleanup_expired_certs_deployment]]
+=== Automatically remove expired SSL/TLS certificates from IAM
+Next, use the link:https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/blob/master/modules/cleanup-expired-certs/README.adoc[`cleanup-expired-certs`]
+module to deploy a Lambda function to remove all expired SSL/TLS certificates from AWS IAM. This module will add all of the relevant IAM resources needed to
+periodically execute the Lambda function and optionally, report metrics to CloudWatch. Be sure to deploy this module into each of your AWS accounts.
+
+[[iam_access_analyzer_deployment]]
+=== Steps to enable IAM Access Analyzer for all enabled AWS regions
+To enable IAM Access Analyzer in your AWS Account or Organization, you need to do it separately for every region.
+
+To enable this for your AWS account:
+* use the examples provided as part of `learning-and-testing/landingzone`:
+link:https://github.com/gruntwork-io/terraform-aws-service-catalog/tree/master/examples/for-learning-and-testing/landingzone/account-baseline-root[account-baseline-root]
+and link:https://github.com/gruntwork-io/terraform-aws-service-catalog/tree/master/examples/for-learning-and-testing/landingzone/account-baseline-security[account-baseline-security].
```

Comment:
@brikis98 @ina-stoyanova I know above and below we've spoken about using Terragrunt examples here in the deployment guide, but I can see you've linked to a Landing Zone example in our service catalog. Also below I noticed you're inserting the actual Terraform module definition. Would it make more sense to rework this like I've done in the Cleanup Expired Certs section and push users down the route of only deploying one module or should we be more clear about requiring users to deploy our account baselines? For example, you could change this section to something along the lines of:

```
If your using Terragrunt, then you can invoke the standalone module directly by following the pattern outlined in
<<wrapper_code_terragrunt>> and using a `terragrunt.hcl` file with the following contents:

.infrastructure-live/<your account>/_global/iam-access-analyzer/terragrunt.hcl
[source,hcl]
----
terraform {
  # Make sure to replace <VERSION> in this URL with the latest terraform-aws-security release
  source = "git::ssh://git@github.com/gruntwork-io/terraform-aws-security.git//modules/iam-access-analyzer-multi-region?ref=<VERSION>"
}

# Include all settings from the root terragrunt.hcl file
include {
  path = find_in_parent_folders()
}

inputs = {
  iam_access_analyzer_type = "ACCOUNT"
  iam_access_analyzer_name = "iam-access-analyzer"
  seed_region              = "us-east-2"
}
----
```
If we went with my suggestion, then I think it would be good to remove any mention of the account baselines just to under complicate the mental model.

Reply:
Ahh, I did have this initially, and then changed it back to a module, because I saw it in other examples. Let's stick to this in that case! I'm happy to change this first thing back on Monday!

Reply:
Done. Apologies - with support and the monthly sync, this slipped my mind, but is done now. Thanks @robmorgan

Reply:
Ok cool!